<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graph</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="plot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>691</width>
      <height>471</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>715</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>&amp;Файл</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionConvert"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>&amp;Свойства</string>
    </property>
    <addaction name="action_delta"/>
    <addaction name="action_temp"/>
    <addaction name="action_report"/>
    <addaction name="action_clear"/>
    <addaction name="action_mincycle"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Открыть преобразованный</string>
   </property>
  </action>
  <action name="actionConvert">
   <property name="text">
    <string>Сконвертировать</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Сохранить график</string>
   </property>
  </action>
  <action name="action_temp">
   <property name="text">
    <string>&amp;Добавить температуру</string>
   </property>
  </action>
  <action name="action_report">
   <property name="text">
    <string>&amp;Отчет</string>
   </property>
  </action>
  <action name="action_delta">
   <property name="text">
    <string>&amp;Задать дельту</string>
   </property>
  </action>
  <action name="action_clear">
   <property name="text">
    <string>О&amp;чистить</string>
   </property>
  </action>
  <action name="action_mincycle">
   <property name="text">
    <string>&amp;Минимальная длина полуцикла</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
